### DISCUSSION TITLE
Welcome to twtcodejam.net Discussions!

### DISCUSSION THREAD

**Discussion by avibn** [Tech-With-Tim/twtcodejam.net#24] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mohamed040406** [comment]
![image](https://user-images.githubusercontent.com/55202038/119339746-d97c1680-bc91-11eb-8184-d26a6f10ec5d.png)
